i18n/en.json

```diff
@@ -2943,6 +2943,8 @@
   "claim.oauth_to_email.pwdNotMatch": "Passwords do not match.",
   "claim.oauth_to_email.switchTo": "Switch {type} to Email and Password",
   "claim.oauth_to_email.title": "Switch {type} Account to Email",
+  "cloud_archived.error.access": "Permalink belongs to a message that has been archived because of cloud plan limits. Upgrade cloud plan to access message again.",
+  "cloud_archived.error.title": "Message Archived",
```

Comment:
nit: Should this say "Post Archived"? I'm not sure what terminology we use with end users & posts

Reply:
At least for limits related things, we've consistently preferred "message" over "post" for end users.